elephant/cubic.py

```diff
@@ -199,23 +197,16 @@ def _kstat(data):
 
     Parameters
     -----
-    data : numpy.aray
+    data : numpy.ndarray
         The population histogram of the population on which are computed
         the cumulants
 
     Returns
     -----
-    kappa : list
-        The first three cumulants of the population count
+    kappa : numpy.ndarray
+        The first three unbiased cumulants of the population count
     '''
-    L = len(data)
-    if L == 0:
+    if len(data) == 0:
         raise ValueError('The input data must be a non-empty array')
-    S = [(data ** r).sum() for r in range(1, 4)]
-    kappa = []
-    kappa.append(S[0] / float(L))
-    kappa.append((L * S[1] - S[0] ** 2) / (L * (L - 1)))
-    kappa.append(
```

Comment:
Note: Verified numerically that both old and new implementation of moments yield `[0.03, 0.2991029910299103, 2.9731431937011825]` with the unit test data.